src/features/CheckoutPage/components/ShippingInformation.jsx

```diff
@@ -0,0 +1,81 @@
+
+import React, {useState} from 'react';
+import {getInfoFromPincode} from '../../../common/utils/api';
+
+const ShippingInformation = ({submitHandler}) => {
+  const [firstName, setFirstName] = useState('');
+  const [lastName, setLastName] = useState('');
+  const [address, setAddress] = useState('');
+  const [apartment, setApartment] = useState('');
+  const [state, setState] = useState('');
+  const [city, setCity] = useState('');
+  const [pincode, setPincode] = useState('');
+
+  const onFormInputChange = async (e) => {
+    if (e.target.name == 'firstName') setFirstName(e.target.value);
```

Comment:
Wrap the Ifs around with curly braces, modern JS may support this, but The Fe Infra dictates that the code should be as easily readable as possible.

Reply:
Coding styles vary from company to company.

Enforced this using ESLINT and fixed :)